src/core/model_config_utils.cc

```diff
@@ -603,17 +603,6 @@ GetNormalizedModelConfig(
     RETURN_IF_ERROR(ReadTextProto(config_path, config));
   }
 
```

Comment:
Here we should normalize platform and backend based on the documentation link I sent. And in validate we should make sure that platform and backend are valid values.